docs/download.md

```diff
@@ -18,18 +18,18 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
 that uses the RAPIDS Accelerator For Apache Spark. See the [getting-started
 guide](https://nvidia.github.io/spark-rapids/Getting-Started/) for more details.
 
-## Release v23.06.0
+## Release v23.08.0
 Hardware Requirements:
 
 The plugin is tested on the following architectures:
 
-	GPU Models: NVIDIA P100, V100, T4 and A2/A10/A30/A100 GPUs
+	GPU Models: NVIDIA P100, V100, T4, A2/A10/A30/A100, L4 and H100 GPUs
 
 Software Requirements:
 
 	OS: Ubuntu 20.04, Ubuntu 22.04, CentOS 7, or Rocky Linux 8
 
-	CUDA & NVIDIA Drivers*: 11.x & v470+
+	NVIDIA Driver*: 470+
 
 	Apache Spark 3.1.1, 3.1.2, 3.1.3, 3.2.0, 3.2.1, 3.2.2, 3.2.3, 3.2.4, 3.3.0, 3.3.1, 3.3.2, 3.4.0 Databricks 10.4 ML LTS or 11.3 ML LTS Runtime and GCP Dataproc 2.0, Dataproc 2.1
```

Comment:
Could you add Databricks 12.2 ML LTS support as well? 
In both FAQ, Get-Started Guide for Databricks, and Download.md.

Reply:
As we've removed Databricks runtime 9.3 support, so we need to update our docs not containing 9.3/spark312db shims, like: 

- https://github.com/NVIDIA/spark-rapids/blob/branch-23.08/docs/dev/shimplify.md `sql-plugin/src/main/312db/scala`
- https://github.com/NVIDIA/spark-rapids/blob/branch-23.08/CONTRIBUTING.md `Databricks Spark based on Spark 3.1.2`

I'm not sure if we can update the above files together in this PR?

Reply:
@NvTimLiu  normally we do not touch those 2 docs. 
Do you have any specific change needed for those 2 docs? Or submit a separate PR to change this?

Reply:
Yes, I can have another separate PR for cleaning up DB 9.3 in our docs.

Reply:
Filed an doc requirement for it: https://github.com/NVIDIA/spark-rapids/issues/8959